.github/workflows/build-publish-docker.yml

```diff
@@ -33,6 +34,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
```

Comment:
[They reckon](https://github.com/docker/build-push-action?tab=readme-ov-file#usage) use this to build arm64 images - not sure how this works with amd64 images though. Could be a trial and error situation.